clients/src/test/java/org/apache/kafka/clients/producer/internals/TransactionManagerTest.java

```diff
@@ -155,6 +155,7 @@ public void testBasicTransaction() throws InterruptedException {
         prepareFindCoordinatorResponse(Errors.NONE, false, FindCoordinatorRequest.CoordinatorType.TRANSACTION, transactionalId);
 
         sender.run(time.milliseconds());  // find coordinator
+        sender.run(time.milliseconds());
```

Comment:
This is probably related to the other change where we don't queue the findCoordinator request by default in `initTransactions`. So we need to do an additional `sender.run` to queue the find coordinator request.